x-pack/plugins/cloud_security_posture/public/pages/findings/translations.ts

```diff
@@ -239,5 +239,5 @@ export const NO_FINDINGS = i18n.translate('xpack.csp.findings.nonFindingsLabel',
 
 export const FINDINGS_SEARCH_PLACEHOLDER = i18n.translate(
   'xpack.csp.findings.searchBar.searchPlaceholder',
-  { defaultMessage: 'Search findings (eg. resource.section : "API Server")' }
+  { defaultMessage: 'Search findings (eg. rule.section.keyword : "API Server" )' }
```

Comment:
do we want to include `keyword` here? AFAIK `keyword` is for `sort`, for plain text search, like this example, omitting the `keyword` should work fine?

Reply:
we have another task to align our findings mapping.
till then, `text` fields allow partial text search, and `keyword` allows exact match.
It also gives you autocompletion while in `text` the user doesn't enjoys that.

so i'm fine with `keyword`